app/main.py

```diff
@@ -1,3 +1,56 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | int | float) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+
+        return Distance(self.km + other)
+
+    def __iadd__(self, other: Distance | int | float) -> Distance:
+        if isinstance(other, Distance):
+            self.km += other.km
+            return self
+
```

Comment:
here you can use else instead 2 returns

Reply:
how?

Reply:
<img width="414" alt="Screenshot 2023-09-08 at 16 43 27" src="https://github.com/mate-academy/py-distance-class/assets/117740672/dc83f9d8-eb55-47fa-8415-f24fcf59ef96">

Reply:
remade